nuxeo-core-binarymanager-s3/pom.xml

```diff
@@ -86,6 +90,21 @@
       <artifactId>nuxeo-runtime-management</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.automation</groupId>
+      <artifactId>nuxeo-automation-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+     <dependency>
+      <groupId>org.nuxeo.ecm.automation</groupId>
+      <artifactId>nuxeo-automation-server</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.automation</groupId>
+      <artifactId>nuxeo-automation-test</artifactId>
+      <type>test-jar</type>
+      <scope>test</scope>
```

Comment:
what is the code from the test scope of nuxeo-automation-test used in nuxeo-core-binarymanager-s3's test scope?

Reply:
this dep is useless
👍 +1